elastic/kibana
# [Security Solution] Use new expandable flyout in alert preview

## Summary

This PR updates the alert preview in Create rule -> Rule preview to use the new expandable alert flyout:

- Switched timeline wrapper to be visible on create rule page. This allows us to keep all the timeline navigation in the new expandable alert flyout
- Disabled alert specific components, when flyout is open in create rule:
   - Alert status is not shown
   - Rule summary preview is disabled
   - Title link to rule details page is removed
   - Exclude filter in/filter out hover actions in highlighted fields
   - New placeholder text for investigation guide and response: we should not show link to documentation when user is setting up a rule

With feature flag on:

https://github.com/elastic/kibana/assets/18648970/a45e930e-f1e8-4899-aef4-1aa0c3dc3330



**How to test**
- Add `xpack.securitySolution.enableExperimental: ['expandableFlyoutInCreateRuleEnabled' ]` to `kibana.yml.dev`
- Go to Rules page -> Detection rules (SIEM) => Create rule
- Pick a rule type and populate the query, click `Continue`
- On the right hand side, click `Refresh`, some alerts should appear in the table
- Click expand on a row

### Checklist
